### DISCUSSION TITLE
Feature Request: Home GUI Changes

### DISCUSSION THREAD

**Discussion by TheBlueKingLP** [Zrips/CMI#1963] [Feature Requests]
**Description of feature/improvement request:**

---
When player use the home gui with /home, they will be able to choose which home to go to, please change it to directly teleporting them instead of executing a command for them, or at least add an option to change what is the command being executed.
What I want is to  remove the "cmi" prefix of the command "/cmi home *home_name* *player*",
so they will use the alias instead.

**Comment by Zrips** [comment]
Why would you want to hide that prefix if its never been shown to the player? Could you explain better where you see that prefix? And we are performing command normally to allow it to pass throw all the needed checks before allow player to teleport to target location. I could remake parts of it, but i wonder why you want to avoid performing commands or hide prefix in general.

**Comment by TheBlueKingLP** [comment]
That is because I am using a special plugin written by myself, what it will do is block all the command except when the command is whitelisted in a list, if I whitelist the command "cmi", the player can try if the "cmi" command exists by just typing "/cmi home"
Also, if you directly teleporting user to the location, it saves a little bit system resources by skipping the "execute the player" to do the home command
